avm/res/azure-stack-hci/cluster/tests/e2e/2nodeswitched.defaults/dependencies.bicep

```diff
@@ -0,0 +1,104 @@
+@minLength(4)
+@maxLength(8)
+param deploymentPrefix string
+param deploymentUsername string
+@secure()
+param deploymentUserPassword string
+param localAdminUsername string
+@secure()
+param localAdminPassword string
+@secure()
+#disable-next-line secure-parameter-default
+param arbDeploymentAppId string = ''
+@secure()
+#disable-next-line secure-parameter-default
+param arbDeploymentSPObjectId string = ''
+@secure()
+param arbDeploymentServicePrincipalSecret string = ''
+param location string
+param clusterNodeNames array
+param softDeleteRetentionDays int = 30
+@minValue(0)
+@maxValue(365)
+param logsRetentionInDays int = 30
+param vnetSubnetId string
+param serviceShort string = 'ashcmin'
+param switchlessStorageConfig bool
+param hciNodeCount int
+param hciVHDXDownloadURL string
+param hciISODownloadURL string
+param clusterWitnessStorageAccountName string
+param keyVaultDiagnosticStorageAccountName string
+param keyVaultName string
+@secure()
+#disable-next-line secure-parameter-default
+param hciResourceProviderObjectId string = ''
+param domainOUPath string?
+param deployArcGateway bool = false
+param hciHostAssignPublicIp bool = false
+
+var arcNodeResourceIds = [
+  for (nodeName, index) in clusterNodeNames: resourceId('Microsoft.HybridCompute/machines', nodeName)
+]
+
+var tenantId = subscription().tenantId
+
+module arcGateway '../../../arc-gateway/main.bicep' = if (deployArcGateway) {
+  name: '${uniqueString(deployment().name, location)}-test-arcgw-${location}-${deploymentPrefix}'
+  params: {
+    location: location
+    name: 'arcg-${location}-${deploymentPrefix}'
+    allowedFeatures: ['*']
+    gatewayType: 'Public'
+  }
+}
+
+module hciHostDeployment '../../../../../../utilities/e2e-template-assets/templates/azure-stack-hci/modules/azureStackHCIHost/hciHostDeployment.bicep' = {
+  name: '${uniqueString(deployment().name, location)}-test-hcihostdeploy-${location}-${deploymentPrefix}'
+  params: {
+    arcGatewayId: deployArcGateway ? arcGateway.outputs.resourceId : null
+    hciHostAssignPublicIp: hciHostAssignPublicIp
+    domainOUPath: domainOUPath
+    deployProxy: false
+    hciISODownloadURL: hciISODownloadURL
+    hciNodeCount: hciNodeCount
+    hostVMSize: 'Standard_E16bds_v5'
+    hciVHDXDownloadURL: hciVHDXDownloadURL
+    localAdminPassword: localAdminPassword
+    location: location
+    switchlessStorageConfig: switchlessStorageConfig
+    vnetSubnetID: vnetSubnetId
+  }
+}
+
+// MICROSOFT GRAPH RESOURCES in Bicep are in preview and break the AVM end-to-end tests
```

Comment:
Right, we should not support experimental features in modules

Reply:
Removed from the module